components/project-details/ProjectDetailsProcess.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Image from 'next/image';
+
+import WrapperStudyDetails from '@/HOC/WrapperStudyDetails';
+import ProjectDetailsHeading from '../reusable/ProjectDetailsHeading';
+import { ProblemStatementProps } from '@/types';
+
+const ProjectDetailsProcess: React.FC<ProblemStatementProps> = ({
+  myProcess,
+}) => {
+  return (
+    <WrapperStudyDetails additionalStyles="bg-white-900 dark:bg-black-200 py-[2.25rem] px-[1.5rem] ">
+      <ProjectDetailsHeading
+        subHeading="Way of work"
+        mainHeading="My Process"
+        additionalStylesMainHeading="pb-[1.5rem]"
+      />
+      <div className="flex flex-wrap justify-between">
+        {myProcess.map((process) => (
+          <div
```

Comment:
This should render a component instead of more <div> elements. This will keep the code clean and more maintainable.